plugins/modules/web_infrastructure/django_manage.py

```diff
@@ -224,20 +261,21 @@ def main():
 
     module = AnsibleModule(
         argument_spec=dict(
-            command=dict(default=None, required=True),
-            app_path=dict(default=None, required=True, type='path'),
-            settings=dict(default=None, required=False),
-            pythonpath=dict(default=None, required=False, aliases=['python_path']),
+            command=dict(required=True, type='str'),
+            app_path=dict(required=True, type='path'),
+            settings=dict(default=None, required=False, type='path'),
+            pythonpath=dict(default=None, required=False, type='path', aliases=['python_path']),
             virtualenv=dict(default=None, required=False, type='path', aliases=['virtual_env']),
 
             apps=dict(default=None, required=False),
-            cache_table=dict(default=None, required=False),
+            cache_table=dict(default=None, required=False, type='str'),
             clear=dict(default=None, required=False, type='bool'),
-            database=dict(default=None, required=False),
+            database=dict(default=None, required=False, type='str'),
             failfast=dict(default=False, required=False, type='bool', aliases=['fail_fast']),
-            fixtures=dict(default=None, required=False),
-            liveserver=dict(default=None, required=False, aliases=['live_server']),
-            testrunner=dict(default=None, required=False, aliases=['test_runner']),
+            fixtures=dict(default=None, required=False, type='str'),
+            liveserver=dict(default=None, required=False, type='str', aliases=['live_server'],
+                            removed_in_version='3.0.0', removed_from_collection='community.general'),
```

Comment:
This deprecation needs a changelog fragment (section `deprecated_features`).

Reply:
done